index.rst

```diff
@@ -1,62 +1,147 @@
+It also touches on what we will here call (for lack of a better term, so far) *Campaign Definition*: the tools and processes for determining the datasets, software, and configuration that will go into a campaign.
+The datasets are our primary concern here, and for these :cite:`DMTN-181` focuses on the concept of exposure *exclusion lists* represented as *data ID sets*.
+A data ID set is a table (at least conceptually) whose columns are data ID keys (butler data dimensions), with each data ID a row; :cite:`DMTN-181` proposes using ``git``-controlled JSON files as the source of truth for these, and uploading them to a table in the `Registry` database for use in `QuantumGraph` generation.
+It also proposes that these be permanent database tables for provenance purposes.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+The processes and tools for creating and maintaining exclusion lists are considered out of scope for campaign management by :cite:`DMTN-181`, but it correctly notes that these must be assembled from many disparate inputs that cannot be assumed to be present in any single database (let along the `Registry` database), so generating exclusion lists from these inputs implicitly as part of the queries that back `QuantumGraph` generation is not viable.
+Defining those processes and tools fully is out of scope for this technote, too, but we will try to anticipate their demands on the middleware.
+That includes support for using the data repository as a place to store and query metrics, dataset annotations, and other criteria that play a role in building exclusion lists, as well as making the `Registry` database the source of truth for curated data ID sets (such as the exclusion lists themselves).
+It should be emphasized that this support has not been requested by those involved in pushing for and designing the Campaign Management system; on the contrary, they regard relying on the `Registry` database as over-centralization and poor separation of concerns.
+These are valid worries - it is fair to say that the butler tries to do too much already, schema stability and migration are not trivial problems, and the middleware codebase currently has few contributors - but if the alternative is standing up, populating, and maintaining another SQL database with content similar to or substantially overlapping the `Registry`'s, then tighter coupling between middleware and Campaign Definition and middleware scope increases seem like the lesser evils.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+.. note::
 
-     Caption text.
+   The Consolidated Database probably has a role to play here; perhaps it is the database that should support Campaign Definition, and perhaps it is even a database that we should consider merging with the `Registry`.
```

Comment:
Consolidated Database means "everything that we want to store in a database" doesn't it? Even for the Exposure table if we integrated it into registry it would mean at least one special table per instrument where the schema is essentially arbitrary since what we can store in it is unlimited and can come from all the fits headers and EFD. At least the ObsCore table is a well-defined schema and is directly related to the Exposure records.

Reply:
> Consolidated Database means "everything that we want to store in a database" doesn't it?

Exactly why I didn't want to try to include it here - as long as that continues to be what it means, it'll continue to be vaporware.  Once it tries to do less we'll have a sense of what it can do.